grpc/execution/server.go

```diff
@@ -208,7 +126,7 @@ func (s *ExecutionServiceServerV1) BatchGetBlocks(ctx context.Context, req *astr
 		block, err := s.getBlockFromIdentifier(id)
 		if err != nil {
 			log.Error("failed finding block with id", id, "error", err)
-			return nil, err
+			return nil, shared.WrapError(err, fmt.Sprintf("failed finding block with id %s", id.String()))
```

Comment:
wrapped error as return type

Reply:
Addressed in https://github.com/astriaorg/flame/pull/30/commits/23a2bd7a275ea8628fc6d53b93665dbe117ac8ce